### PR TITLE
Include a note in the CHANGELOG about how to lint with Swift 3.0

### PR DESCRIPTION
**Pull request by orta**
re https://github.com/CocoaPods/CocoaPods/issues/5875

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,7 +22,8 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Enhancements
 
-* Use the SWIFT_VERSION when linting pods.  
+* Use the SWIFT_VERSION when linting pods. To lint with Swift 3.0
+  add a Swift Version file. `echo "3.0" >> .swift_version`.
```
Comment (mgadda) [inline CHANGELOG.md]: I think this should say `.swift-version` not `.swift_version`, at least based on my empirical evidence.

Comment (orta) [inline CHANGELOG.md]: it was fixed in a later PR

Comment (mgadda) [inline CHANGELOG.md]: Ah, sorry I didn't notice that. Thanks!

Comment (i-schuetz) [reply CHANGELOG.md]: Doesn't seem to work for Swift 4.2

**Pull request merged**